From 4bd30463b7ee6a06dfe310c1d89cf1e060232d41 Mon Sep 17 00:00:00 2001
From: Lukas Czerner <lczerner@redhat.com>
Date: Thu, 23 Oct 2014 09:54:14 -0400
Subject: [fs] ext4: delete "set but not used" variables

Message-id: <1414058125-4183-101-git-send-email-lczerner@redhat.com>
Patchwork-id: 98199
O-Subject: [RHEL 7.1 PATCH v2 100/171] ext4: delete "set but not used" variables
Bugzilla: 1150139
RH-Acked-by: Eric Sandeen <sandeen@redhat.com>

From: jon ernst <jonernst07@gmail.com>

BZ 1150139
https://bugzilla.redhat.com/show_bug.cgi?id=1150139

BREW 8142710
https://brewweb.devel.redhat.com/taskinfo?taskID=8142710

Upstream commit d7092ae2973f20a39fee786c47e5edf18ced088f

 Signed-off-by: Jon Ernst <jonernst07@gmail.com>
 Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>
 Reviewed-by: Zheng Liu <wenqing.lz@taobao.com>
 Signed-off-by: Lukas Czerner <lczerner@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/ext4/inline.c b/fs/ext4/inline.c
index 43648fe..7ca111a 100644
--- a/fs/ext4/inline.c
+++ b/fs/ext4/inline.c
@@ -1873,7 +1873,6 @@ int ext4_try_to_evict_inline_data(handle_t *handle,
 {
  int error;
  struct ext4_xattr_entry *entry;
- struct ext4_xattr_ibody_header *header;
  struct ext4_inode *raw_inode;
  struct ext4_iloc iloc;
 
@@ -1882,7 +1881,6 @@ int ext4_try_to_evict_inline_data(handle_t *handle,
   return error;
 
  raw_inode = ext4_raw_inode(&iloc);
- header = IHDR(inode, raw_inode);
  entry = (struct ext4_xattr_entry *)((void *)raw_inode +
          EXT4_I(inode)->i_inline_off);
  if (EXT4_XATTR_LEN(entry->e_name_len) +
diff --git a/fs/ext4/ioctl.c b/fs/ext4/ioctl.c
index 60589b6..6bea806 100644
--- a/fs/ext4/ioctl.c
+++ b/fs/ext4/ioctl.c
@@ -101,9 +101,8 @@ static long swap_inode_boot_loader(struct super_block *sb,
  handle_t *handle;
  int err;
  struct inode *inode_bl;
- struct ext4_inode_info *ei;
  struct ext4_inode_info *ei_bl;
- struct ext4_sb_info *sbi;
+ struct ext4_sb_info *sbi = EXT4_SB(sb);
 
  if (inode->i_nlink != 1 || !S_ISREG(inode->i_mode)) {
   err = -EINVAL;
@@ -115,9 +114,6 @@ static long swap_inode_boot_loader(struct super_block *sb,
   goto swap_boot_out;
  }
 
- sbi = EXT4_SB(sb);
- ei = EXT4_I(inode);
-
  inode_bl = ext4_iget(sb, EXT4_BOOT_LOADER_INO);
  if (IS_ERR(inode_bl)) {
   err = PTR_ERR(inode_bl);
-- 
1.7.1